scripts/mixins/zones/Abyssea-Altepa.lua

```diff
@@ -0,0 +1,7 @@
+---------------------------------------------------------
+
+---------------------------------------------------------
+mixins = 
+{
+    require("scripts/mixins/drop_lights"),
```

Comment:
https://discordapp.com/channels/639659267003252746/639670779386134548/745655082934337536

Reply:
do you want to handle these last two comments as an issue?

Reply:
nrly..